### PR TITLE
Remove PHP<7 leftovers

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

Forgot while cleaning up

### CODE REVIEW

File: src/Symfony/Component/PropertyAccess/PropertyAccessor.php [hunk 1]
```diff
@@ -240,11 +240,6 @@ public function setValue(&$objectOrArray, $propertyPath, $value)
             }
         } catch (\TypeError $e) {
             self::throwInvalidArgumentException($e->getMessage(), $e->getTrace(), 0);
-        } finally {
-            if (PHP_VERSION_ID < 70000 && false !== self::$previousErrorHandler) {
-                restore_error_handler();
-                self::$previousErrorHandler = false;
```
Comment (stof) [inline src/Symfony/Component/PropertyAccess/PropertyAccessor.php]: do we still need `self::$previousErrorHandler` ?

Comment (chalasr) [inline src/Symfony/Component/PropertyAccess/PropertyAccessor.php]: no indeed, removed `self::$errorHandler`and `handleError()` which were unused also

**Comment by chalasr** [comment]
build failure unrelated

File: src/Symfony/Component/Serializer/Tests/Encoder/JsonEncodeTest.php [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
 class JsonEncodeTest extends TestCase
 {
-    private $encoder;
+    private $encode;
```
Comment (xabbuh) [inline src/Symfony/Component/Serializer/Tests/Encoder/JsonEncodeTest.php]: should probably be done in older branches too

**Review by xabbuh — APPROVED** [review]

**Review by ogizanagi — APPROVED** [review]

File: src/Symfony/Component/Serializer/Tests/Encoder/JsonEncodeTest.php (hunk 2 above)
Comment (nicolas-grekas) [inline src/Symfony/Component/Serializer/Tests/Encoder/JsonEncodeTest.php]: and we should rename to encoder everywhere also

File: src/Symfony/Component/PropertyAccess/PropertyAccessor.php (hunk 1 above)
Comment (nicolas-grekas) [inline src/Symfony/Component/PropertyAccess/PropertyAccessor.php]: when was self::$previousErrorHandler set to something else than false?

File: src/Symfony/Component/Serializer/Tests/Encoder/JsonEncodeTest.php (hunk 2 above)
Comment (chalasr) [inline src/Symfony/Component/Serializer/Tests/Encoder/JsonEncodeTest.php]: hmm, altough I find the name weird, the tested class is JsonEncode, not JsonEncoder

File: src/Symfony/Component/PropertyAccess/PropertyAccessor.php (hunk 1 above)
Comment (chalasr) [inline src/Symfony/Component/PropertyAccess/PropertyAccessor.php]: there https://github.com/symfony/symfony/commit/7091fb4df5c73413f8a6d490bf0f738bc39dfeb9#diff-3e45cf556c0330623c0167841d303dffL198

**Comment by nicolas-grekas** [comment]
Thank you @chalasr.

**Pull request merged**